### PR TITLE
Repeated css

### PR DESCRIPTION
**Pull request by Vidmapps**
Is it good so far? I am not sure if doing it correctly. No rush, take your time to check on Friday

### CODE REVIEW

File: partials.css [hunk 1]
```diff
@@ -30,6 +30,52 @@
     position: absolute;
 }
 
+.bColor {
```
Comment (alynioke) [inline partials.css]: i recommend putting full name for all variables - not `bColor` but `backgroundColor`. This one i do understand but next one - `fColor` - i already don't get what `f` stands for. It's always better to be more explicit in naming. Don't be afraid to make too long names!

File: partials.css [hunk 2]
```diff
@@ -30,6 +30,52 @@
...
+    background-color: #212938;
+}
+
+.fColor {
```
Comment (alynioke) [inline partials.css]: I see you're using camelCase in this file, but in `style.css` you're using kebab-case. It has to be only one - so please choose which you like better and refactor the rest

File: partials.css [hunk 3]
```diff
@@ -30,6 +30,52 @@
...
+
+.borderRadius {
+    border-radius: 4px;
+}
+
+.smallerFont {
+    font-size: 16px;
+}
+
+.collumnNameFontSize {
```
Comment (alynioke) [inline partials.css]: here and line 77:
`collumn` -> `column` (one `l`)

File: partials.css (hunk 1 above)
Comment (Vidmapps) [reply partials.css]: Should be ok now

File: partials.css (hunk 2 above)
Comment (Vidmapps) [reply partials.css]: Kebabed everything 🍡

File: partials.css (hunk 3 above)
Comment (Vidmapps) [reply partials.css]: Typos fixed 🔢

**Pull request merged**